readme.txt

```diff
@@ -1,3 +1,5 @@
-This repository contains a Meson build definition for project eigen.
+This repository contains a Meson build definition for project eigen3.
 
-For more information please see http://mesonbuild.com.
+All files in this repository have the same license as the original project.
```

Comment:
This is no longer true, please revert to the original file from master. We [decided to convert all wraps to MIT license](https://github.com/mesonbuild/wrapweb/issues/28) regardless of what project ships.